x-pack/plugin/ccr/qa/multi-cluster-with-non-compliant-license/src/test/java/org/elasticsearch/xpack/ccr/CcrMultiClusterLicenseIT.java

```diff
@@ -31,14 +31,6 @@ protected boolean preserveClusterUponCompletion() {
         return true;
     }
 
-    public void testResumeFollow() {
```

Comment:
The resume api now looks up the follow index metadata in order to get the leader index name and leader cluster name and because of that this fails with a different error. We can't execute a put follow to create the follow index, because the leader cluster runs with a basic license. So I don't see a way how to test this in another way.